app-emulation/FEX/FEX-2409-r2.ebuild

```diff
@@ -45,6 +45,8 @@ BDEPEND="
 	thunks? ( !crossdev-toolchain? (
 		sys-devel/x86_64-multilib-toolchain
 	) )
+	app-emulation/fex-rootfs-gentoo
+	app-emulation/fex-rootfs-mesa-asahi
```

Comment:
Since the rootfs package depends on systemd, we should gate it behind a systemd use flag

Reply:
Changed the base layers to conditionally depend on systemd, and print a message that the user should really use systemd or if they really do not want it - assemble all the pieces themselves. Should be safe to unconditionally depend now.